execution/executor/src/components/chunk_output.rs

```diff
@@ -50,17 +48,14 @@ impl ChunkOutput {
             transactions_and_outputs.into_iter().unzip();
 
         // collect all accounts touched and dedup
-        let state_updates = transaction_outputs
-            .iter()
-            .flat_map(|o| o.write_set())
-            .collect::<HashSet<_>>();
+        let write_set = transaction_outputs
+            .clone()
+            .into_iter()
+            .map(|o| o.into().0)
+            .collect::<Vec<_>>();
```

Comment:
why?

Reply:
to reuse prime_cache_by_write_set func

Reply:
why do we need clone and call `into`. I think you can just change the `HashSet` to `Vec`

Reply:
Done